wordpress-mobile/WordPress-iOS
# My Sites: Show/hide search bar based visible sites count.

Fixes #8522 

To test:
- Have at least 4 sites visible.
  - Verify the search bar appears.
- Have less than 4 sites visible.
  - Verify the search bar _does not_ appear.

@frosty  - could I bother you for a review please? Thanks!!
